spec.html

```diff
@@ -32989,13 +32987,11 @@
 
       <!-- es6num="24.1.2.1" -->
       <emu-clause id="sec-arraybuffer-length">
-        <h1>ArrayBuffer ( _length_ )</h1>
-        <p>ArrayBuffer called with argument _length_ performs the following steps:</p>
+        <h1>ArrayBuffer ( [ _length_ ] )</h1>
```

Comment:
Object has an optional argument and an extraordinary length value. https://tc39.github.io/ecma262/#sec-object-value

We may follow this and define an extraordinary length value for ArrayBuffer.

String, Number and Boolean constructors could be revisited to follow Object and (many other objects) declaring the optional argument.

TBQH, I believe classic constructors as Array, Object and especially Date inherits the length value from the previous definition where it accounted all the possible arguments. The length value is not critical for average use of JS, but this could be discussed on another topic, and eventually fixed with a more consistent definition.